From bddb53abdd1a77b1c019457f7eea4a6404a5e492 Mon Sep 17 00:00:00 2001
From: Andre Heider <a.heider@gmail.com>
Date: Sun, 17 Jan 2021 11:42:57 +0100
Subject: [PATCH] meson: stop checking for libwine

From the WINE 6.0 release notes:
The libwine library is no longer used, but it's still provided to
support running Winelib modules built against older versions. It is
deprecated however, and will be removed in a future Wine release.

libwine wasn't used anymore anyway, this just ensures that this tree can
be built against future WINE versions.
---
 meson.build | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/meson.build b/meson.build
index 0969777..47ee51a 100644
--- a/meson.build
+++ b/meson.build
@@ -35,12 +35,10 @@ code = '''#ifndef __WINE__
 #error 1
 #endif'''
 
-if not cc.compiles(code, name: 'winelib check')
+if not cc.compiles(code, name: 'winegcc check')
   error('A WINE cross compiler is required')
 endif
 
-dep_wine = cc.find_library('wine')
-
 if not cc.has_header_symbol('windows.h', 'wine_get_unix_file_name')
   error('WINE headers not found')
 endif
